### DISCUSSION TITLE
Victron HEX-Mode protocol: use case?

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#1107] [General]
Hi community, now that there's support for Victrons HEX-Mode protocol available, am I missing out something valuable if using only TEXT-Mode?
A real life scenario would be great. Cheers!

**Comment by SW-Niko** [comment]
Hello @greymda ,

copy from the wiki...
Note: Since firmware version 2024.05.03, OpenDTU-OnBattery supports Victrons HEX-Mode protocol to retrieve the “Total DC input power” and the “Charger internal temperature” from the MPPT. Check the chapter “Protocol Variants” below for more details.

There are 2 ways to get the "Total DC input power” (used for solar-passthough) all MPPTs must be connected over VE.Direct (Text-Mode) or one MPPT must be connected over VE.Direct (Hex-Mode) and all MPPT must be connected over VE.Smart Network.

![grafik](https://github.com/user-attachments/assets/c11f82e1-be0f-4f1a-a959-558272f0180b)

**Reply by greymda** [reply]
oh, so it makes sense only if using several Victron MPPTs?

**Reply by SW-Niko** [reply]
Yes, 
but to be honest ... not sure if someone added additional features in the meantime.
